edweiss-app/app/(app)/courses/[id]/index.tsx

```diff
@@ -154,42 +145,34 @@ const CoursePage: ApplicationRoute = () => {
 				{upcomingAssignments.length > 0
```

Comment:
You can use the `For` component of the standard library for a cleaner code where this logic is handled for you.